### PR TITLE
Implementing sign out button functionality

### PR DESCRIPTION
**Pull request by mlcvijetinovic**
Added functionality to sign out button.We now sign out from seasion,not
redirected to login page.This is achived with on load function,which
passing google object to redirected page,and we are able to sign out
from dashboard and login page.Also,added new client id for google
account.

Added new internationalization to some functionality.It's implemented on
registration panel (welcome message and redirect label info).

Added translations to .en and .de properties.

Added in documentation solution for line feed problem.

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/jsp/Login.jsp [hunk 1]
```diff
@@ -20,9 +20,9 @@
       <div class="row" style="margin-top: 30px;" align="center">
         <div id="registerBT"
           style="margin-bottom: 40px; margin-top: 80px; height: 200px; width: 250px; display: none; visibility: hidden;">
-          <h4 id="welcome" style="font-family: Roboto, sans-serif; font-weight: 700;"></h4>
+          <h4 id="welcome" style="font-family: Roboto, sans-serif; font-weight: 700;"><spring:message code="user.welcome.message"/></h4>
           <h4 id="welcome2" style="font-family: Roboto, sans-serif; font-weight: 700; margin-bottom: 25px;"></h4>
-          <button id="registerBT2" class="btn btn-success btn-lg">Register with Google</button>
+          <button id="registerBT2" class="btn btn-success btn-lg">Register with google</button>
```
Comment (stanislavgrujic) [inline src/main/webapp/WEB-INF/views/jsp/Login.jsp]: Shouldn't this also be internationalized?

File: src/main/webapp/resources/core/js/login.js [hunk 2]
```diff
@@ -14,13 +23,15 @@ function onSignIn(googleUser) {
...
   auth2.signOut().then(function() {
+    window.location.href = "http://localhost:8080";
   });
 });
 
 $("#registerBT2").on(
     "click",
     function() {
       var user = {};
+      $("#registerBT2").text('Loading...');
```
Comment (stanislavgrujic) [inline src/main/webapp/resources/core/js/login.js]: I18N?

File: src/main/webapp/resources/core/js/login.js [hunk 3]
```diff
@@ -54,7 +64,7 @@ function checkUserById() {
               + profile.getId()
         }, 3000);
         $("#registerBT2").prop('disabled', true);
-        $("#registerBT2").text("Redirecting...");
+        $("#registerBT2").text('Redirecting...');
```
Comment (stanislavgrujic) [inline src/main/webapp/resources/core/js/login.js]: I18N?

File: src/main/webapp/resources/core/js/login.js [hunk 4]
```diff
@@ -1,10 +1,23 @@
 var profile;
+var auth2;
+var IsLoggedIn = false;
+var dataValue;
+
+function onLoad() {
+  if (!IsLoggedIn) {
+    gapi.load('auth2', function() {
+      gapi.auth2.init({
+        client_id : '285356479645-dfh2hqjs73ba32mqshfgouil2btm82um.apps.googleusercontent.com'
```
Comment (BezbojanRS) [inline src/main/webapp/resources/core/js/login.js]: Hold important IDs in constants. 
It is easier to change, explain and use them.

File: src/main/webapp/resources/core/js/login.js [hunk 5]
```diff
@@ -14,48 +27,55 @@ function onSignIn(googleUser) {
...
+  user.email = profile.getEmail();
+  var xhr = new XMLHttpRequest();
+  xhr.open("POST", 'http://localhost:8080/user/add');
+  xhr.setRequestHeader('Content-Type', 'application/json; charset=UTF-8');
+  xhr.send(JSON.stringify(user));
+  xhr.onreadystatechange = function() {
+    if (this.readyState == 4 && xhr.status == 200) {
+      setTimeout(function() {
+        window.location = "http://localhost:8080/dashboard/user/" + user.googleId
+      }, 3000);
```
Comment (BezbojanRS) [inline src/main/webapp/resources/core/js/login.js]: Why you are waiting for 3sec?

File: src/main/webapp/resources/core/js/login.js [hunk 6]
```diff
@@ -14,48 +27,55 @@ function onSignIn(googleUser) {
...
 }
+
+function onEvent(id, dataValue) {
+  var translate = $("#registerBT2").attr(dataValue);
+  $(id).text(translate);
+}
+
+function onReload(id, dataValue) {
+  var translate = $("#registerBT2").attr(dataValue);
+  $(id).text(translate);
```
Comment (BezbojanRS) [inline src/main/webapp/resources/core/js/login.js]: Duplicated code in onEvent and onReload

File: src/main/webapp/resources/core/js/login.js (hunk 5 above)
Comment (mlcvijetinovic) [inline src/main/webapp/resources/core/js/login.js]: We use it, to show user welcome message,and verification that the user has successfully registered.This 3 seconds (3000 millseconds) slow down redirection.

**Comment by stanislavgrujic** [comment]
Please resolve merge conflicts.

**Pull request merged**